styles/blocks/_gallery.scss

```diff
@@ -1,255 +1,24 @@
-$fotorama-duration-time: 0.3s;
-$fotorama-arw-size: 95px;
-$fotorama_close_size: 30px;
-$size-fotorama-block: 50px;
-$fotorama-thumb-arrow: 30px;
-
-@mixin fotorama-abs-center() {
-    transform: translate3d(-50%, -50%, 0);
-    left: 50%;
-    position: absolute;
-    top: 50%;
-}
-
-@mixin fotorama-shadow-gradient($x, $y) {
-    background-image: linear-gradient(transparent, rgba(0, 0, 0, 0.2) 25%, rgba(0, 0, 0, 0.3) 75%, transparent), radial-gradient(farthest-side at $x $y, rgba(0, 0, 0, 0.4), transparent);
-}
-
-@mixin fotorama-arrow-gradient($d) {
-    background-image: linear-gradient($d, rgba(255, 255, 255, 0.7), rgba(255, 255, 255, 0));
-}
-
-@mixin fotorama-stretch() {
-    bottom: 0;
-    height: 100%;
-    left: 0;
-    position: absolute;
-    right: 0;
-    top: 0;
-    width: 100%;
-}
-
-.fotorama-print-background {
-    @media print {
-        background: none;
-    }
-}
-
-.fotorama-sprite {
-    @extend .fotorama-print-background;
-    background: url('../images/gallery.png') no-repeat;
-}
-
-.fotorama-inline-block {
-    display: inline;
-    zoom: 1;
-    display: inline-block;
-    vertical-align: middle;
-}
-
-.fotorama__zoom-in,
-.fotorama__zoom-out {
-    display: none;
-}
-
-.fotorama__fullscreen {
-    .fotorama__zoom-in,
-    .fotorama__zoom-out {
-        position: absolute;
-        width: 50px;
-        height: 50px;
-        display: block;
-        margin-left: 20px;
-        z-index: 1000;
-    }
-
-    .fotorama__zoom-out {
-        @extend .fotorama-sprite;
-        top: 51px;
-        background-position: 0 (-$size-fotorama-block);
-    }
-
-    .fotorama__zoom-in {
-        @extend .fotorama-sprite;
-        top: 0;
-        background-position: 0 0;
-    }
-}
-
-.fotorama__zoom-in,
-.fotorama__zoom-out {
-    display: none;
-}
-
-.fotorama__fullscreen {
-    .fotorama__zoom-in,
-    .fotorama__zoom-out {
-        position: absolute;
-        width: 50px;
-        height: 50px;
-        display: block;
-        margin-left: 20px;
-        cursor: pointer;
-    }
-
-    .fotorama__zoom-out {
-        @extend .fotorama-sprite;
-        top: 51px;
-        background-position: 0 (-$size-fotorama-block);
-    }
+// /**
+// * Copyright © 2016 Magento. All rights reserved.
+// * See COPYING.txt for license details.
+// */
 
-    .fotorama__zoom-in {
-        @extend .fotorama-sprite;
-        top: 0;
-        background-position: 0 0;
-    }
-}
+@import 'gallery/variables';
+@import 'gallery/mixins';
+@import 'gallery/extends';
+@import 'gallery/focus';
+@import 'gallery/fullscreen';
 
 .fotorama__zoom-in,
 .fotorama__zoom-out {
     display: none;
```

Comment:
Sure, but I followed default Magento LESS structure so it'll be easy later to track changes and update blank theme and child themes